devito/operator/operator.py

```diff
@@ -603,6 +603,39 @@ def cfunction(self):
 
         return self._cfunction
 
+    def cinterface(self, force=False):
```

Comment:
How about adding name=None to the API (and a `if not name` before line 621) so that the caller can change the default?

Reply:
the user can control that already via `Operator(..., name=X)` -- so here (unlike "classic" devito use model) we don't use the `.soname` but rather the actual `Operator` name.